rasa/shared/core/training_data/story_writer/yaml_story_writer.py

```diff
@@ -263,6 +244,25 @@ def process_user_utterance(
                     entities.append(entity["entity"])
             result[KEY_ENTITIES] = entities
 
+        if hasattr(user_utterance, "inline_comment"):
+            comment = user_utterance.inline_comment(force_add=not entities)
```

Comment:
Looks great 💯 I think my only suggestion would be to rename this argument into something that would clarify instantly what its purpose is.

Reply:
Tried to find one, I think it suits better than the previous one.